echopype/convert/set_groups_ek60.py

```diff
@@ -611,6 +611,9 @@ def set_beam(self) -> xr.Dataset:
             [ds, xr.merge(ds_backscatter)], combine_attrs="override"
         )  # override keeps the Dataset attributes
 
+        # manipulate Dataset to adhere to convention
```

Comment:
I suggest a more fleshed out comment. Otherwise it sounds generic. Say: 

"Manipulate some Dataset dimensions to adhere to convention"